index.js

```diff
@@ -61,15 +61,14 @@ function convertCrontabs() {
   for (const funcName in this.serverless.service.functions) {
     for (const eventIndex in this.serverless.service.functions[funcName]
       .events) {
-      const event = this.serverless.service.functions[funcName].events[
+      const schedule = this.serverless.service.functions[funcName].events[
         eventIndex
       ];
       // only process events with a schedule & a timezone
       if (
-        event.hasOwnProperty("schedule") &&
-        event.schedule.hasOwnProperty("timezone")
+        schedule.hasOwnProperty("rate") &&
```

Comment:
I don't think this is the case anymore
https://www.serverless.com/framework/docs/providers/aws/events/schedule